compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1926,6 +1926,8 @@ supported_targets! {
     ("mipsel-sony-psp", mipsel_sony_psp),
     ("mipsel-sony-psx", mipsel_sony_psx),
     ("mipsel-unknown-none", mipsel_unknown_none),
+    ("mips-mti-none-elf", mips_mti_none_elf),
+    ("mipsel-mti-none-elf", mipsel_mti_none_elf),
```

Comment:
It's not quite clear to me how this macro invocation's entries are grouped, but I guess this is as good place as any

Reply:
Yes. We should have better grouping in future.